src/features/apiDelegation/offered/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -196,81 +198,79 @@ export const ChooseOrgPage = () => {
       <Page size={isSm ? PageSize.Small : PageSize.Medium}>
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.give_access_to_new_api')}</PageHeader>
         <PageContent>
-          <div className={common.pageContent}>
-            <h2 className={classes.topText}>{t('api_delegation.new_org_content_text')}</h2>
-            {isSm && chosenItems.length > 0 && (
-              <div className={classes.chosenOrgs}>
-                <h4 className={classes.chosenOrgsHeader}>
-                  {t('api_delegation.businesses_going_to_get_access')}
-                </h4>
-                <div className={classes.actionBarScrollContainer}>{chosenItems}</div>
-              </div>
-            )}
-            <div className={classes.searchSection}>
-              <SearchField
-                label={String(t('api_delegation.search_for_buisness'))}
-                value={searchString}
-                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-                  handleSearch(event.target.value);
-                }}
-                aria-label={String(t('api_delegation.search_for_buisness'))}
-              ></SearchField>
+          <h2 className={classes.topText}>{t('api_delegation.new_org_content_text')}</h2>
```

Comment:
What's these changes about?

Reply:
Like the other one, I removed an unused div so the indents changed.
I think that's it